docker-compose.yml

```diff
@@ -1,6 +1,13 @@
+# We can not extend services with liks and we don't want it,
+# overall, the app image is exactly the same.
```

Comment:
Oh, should probably remove it. It is related to the use of `image: aleph_app` below in the services.